core/arch/arm/include/kernel/generic_boot.h

```diff
@@ -34,9 +34,14 @@ void arm_cl2_config(vaddr_t pl310);
 void arm_cl2_enable(vaddr_t pl310);
 
 #if defined(CFG_BOOT_SECONDARY_REQUEST)
-extern paddr_t ns_entry_addrs[];
+/* align to largest expected cache line size */
+struct ns_entry_context {
+	paddr_t entry_point;
+	uint32_t r0;
+} __aligned(64);
```

Comment:
Please use a define instead of the magic number `64`, `STACK_ALIGNMENT` seems to be the right thing even if the name is a little bit off.

Reply:
The alignment attribute isn't needed if we don't need to flush the cache, which we don't.